Wire-iOS/Sources/UserInterface/Conversation/Content/Cells/ConfigurationMessageCell/ConversationMessageSectionController.swift

```diff
@@ -169,6 +171,28 @@ final class ConversationMessageSectionController: NSObject, ZMMessageObserver {
         return [AnyConversationMessageCellDescription(locationCell)]
     }
 
+    private var addCompositeMessageCells: [AnyConversationMessageCellDescription] {
+        guard let compositeMessage = message as? ConversationCompositeMessage else { return [] }
+        
+        var cells: [AnyConversationMessageCellDescription] = []
+        
+        compositeMessage.compositeMessageData?.items.forEach() { item in
+            switch item {
+            case .text(let data):
+                let textCells = ConversationTextMessageCellDescription.cells(textMessageData: data, message: message, searchQueries: context.searchQueries)
```

Comment:
IMHO I would indent more like:

```suggestion
                ConversationTextMessageCellDescription.cells(textMessageData: data,
                                             message: message,
                                             searchQueries: context.searchQueries)```